packages/ember-routing-htmlbars/lib/keywords/action.js

```diff
@@ -7,7 +7,7 @@ import { keyword } from 'htmlbars-runtime/hooks';
 import closureAction from 'ember-routing-htmlbars/keywords/closure-action';
 
 /**
-  The `{{action}}` helper provides a way to pass triggers for behavior (usually
+  The `{{action` helper provides a way to pass triggers for behavior (usually
```

Comment:
I'm personally OK with `{{action}}`/`{{action` in prose. This is consistent with current use in the guides, as well as in other sites (like MDN when it refers to HTML elements, ex. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe).

If the current trend is to use opening and closing double-curlies like `{{action}}`, I'm fine with using that here. And the more I look at the guides and other doc pages, the more I'm convinced that this is the case.